doc/whats_new.rst

```diff
@@ -121,6 +121,8 @@ Changelog
 
 Bug
 ~~~
+- Fix :meth:`mne.io.brainvision.brainvision._read_annotations_brainvision` to accomodate vmrk files which do not have any annotations by `Alexander Kovrig`_
```

Comment:
don't document changes to private functions. Use the link to the public functions that will be fixed. thanks